src/main/scala/com/ing/wbaa/rokku/sts/api/AdminApi.scala

```diff
@@ -62,4 +64,26 @@ trait AdminApi extends LazyLogging with Encryption {
     }
   }
 
+  def enableOrDisableAccount: Route =
+    put {
+      path("account" / Segment / ("enable" | "disable")) { account =>
+        authorizeToken(verifyAuthenticationToken) { keycloakUserInfo =>
+          extractUri { uri =>
+            if (userInAdminGroups(keycloakUserInfo.userGroups)) {
+              val action = uri.path.toString.split("/").last match {
+                case "enable"  => true
+                case "disable" => false
```

Comment:
matching error will be thrown in other value is set

Reply:
it shouldn't be since path matcher will only match two cases